packages/twitch-chat-client/src/ChatClient.ts

```diff
@@ -452,6 +452,9 @@ export default class ChatClient extends IRCClient {
 	private readonly _onIntermediateAuthenticationFailure: (
 		handler: (message: string) => void
 	) => Listener = this.registerEvent();
+	private readonly _onMessageRatelimit: (
```

Comment:
You want to actually use this, right? Please remove the `private` keyword, the underscore, and move it up to the other public events. Also please document it.